examples/bpk-component-section-header/examples.module.scss

```diff
@@ -0,0 +1,33 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index.scss';
+
+.bpk-section-header-examples {
+  &--on-dark {
```

Comment:
If this is the first level, could use `&__on-dark` like many other cases in backpack.

Reply:
Thank you for reviews, `&--on-dark` is acceptable like [BpkModalInner.module.scss](https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-modal/src/BpkModalInner.module.scss#L52)